docs/the-guide/chapter-9-cleaning-up-ressources-and-environment/index.md

```diff
@@ -0,0 +1,110 @@
+# Cleaning up resources and environments
+
+Now that you have completed the guide, it's important to clean up the resources and environments you created to avoid incurring unnecessary costs. Here's a step-by-step guide to help you do that.
+
+!!! warning
+    If you are using a new google cloud account, make sure to delete the resources you created before the credits are consumed. Otherwise, **you will be charged for the resources you created.**
+
+
+## Steps
+
+
+### Google Cloud
+
+#### Delete the Google Storage bucket
+
+!!! warning
+    Ignore this if you plan deleting the git repository. When deleting the Google Storage bucket, **it will break the DVC remote**. You will need to reconfigure it to a new remote if you want to keep the git repository to use DVC.
+
+To delete the Google Storage bucket you created:
+
+1. Go to the [Google Cloud Storage Console](https://console.cloud.google.com/storage){:target="_blank"}.
+2. Make sure you selected the correct project.
+3. Select the bucket you want to delete from the bucket list.
+4. Click on **Delete** at the top of the page.
+5. Follow the instructions to delete the bucket.
+
+#### Delete the Service Account
+
+To delete the service account you created:
```

Comment:
Is it possible to use the Google Cloud CLI (gcloud) so we don't have to go through the UI?